.dotnet/src/Custom/Models/OpenAIModelModelFactory.cs

```diff
@@ -5,7 +5,7 @@
 namespace OpenAI.Models;
 
 /// <summary> Model factory for models. </summary>
-public static partial class OpenAIModelsModelFactory
+public static partial class OpenAIModelModelFactory
```

Comment:
This one should be OpenAIModelsModelFactory because we're using the name of the namespace "OpenAI.Models" (I acknowledge that ModelsModelFactory sounds silly 🙃)